frb_example/benches/README.md

```diff
@@ -0,0 +1,44 @@
+## Benchmarks example for `flutter_rust_bridge`
+
+*`flutter_rust_bridge` or "`FRB`"" for conciseness.*
+
+This repo is a sample on how to implement benchmarks with FRB.
+
+### Get runnin'
+
+#### simple benchmark
+
+First example uses [benchmark_harness](https://pub.dev/packages/benchmark_harness), in `dart/lib/benchmark`.
+
+It can be run like:
+
+```sh
+just bench
+# just bench --define=SILICON=true (on latest MacOS Silicon)
+```
+
+:bulb: You can also customize the size of the vecs benchmarked with env var e.g. `SAMPLE_COUNT=100`. (default: `1000`)
+
+Example resides in [frb_example/benches/dart/lib/benchmark](https://github.com/fzyzcjy/flutter_rust_bridge/tree/master/frb_example/benches/dart/lib/benchmark).
+
+#### custom benchmark
+
+In the same folder, you'll also find a slightly more convoluted example on how to implement a custom platform-agnostic interceptor.
+
+It can be run like:
+
+```sh
+just test-benches
```

Comment:
nit: well, I feel `just test-benches` vs `just bench` like the former is to test the latter, instead of "custom" vs "simple". Maybe we can rename it?

Reply:
good idea, I'm on it.